xbmc/music/Artist.h

```diff
@@ -29,6 +29,15 @@ class CDiscoAlbum
   std::string strReleaseGroupMBID;
 };
 
+class CArtistVideoLinks
+{
+public:
+  std::string strTitle;
```

Comment:
Please avoid Hungarian notation. Since all members are public you likely want a struct instead of a class?
If not it's better to create the necessary getters/setters :)